<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next_frame">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_frame">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>480</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="video_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>720</width>
      <height>405</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_pos">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="set_cut_out">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CutOut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_cut_in">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CutIn</string>
    </property>
   </widget>
   <widget class="QLabel" name="cut_in_pos">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cut_out_pos">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>540</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_frame_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;</string>
    </property>
    <property name="shortcut">
     <string>PgUp</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="next_frame_2">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>480</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
    <property name="shortcut">
     <string>PgDown</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="position_slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>721</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="go_cut_in">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go CutIn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go_cut_out">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go CutOut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_media_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="next_media_file">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="current_media_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_cut">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Cut</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="next_cut">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_cut">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="current_cut">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_cut">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Cut</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="jump_to_frame">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="next_frame_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
    <property name="shortcut">
     <string>Down</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_frame_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
    <property name="shortcut">
     <string>Up</string>
    </property>
    <property name="autoRepeatInterval">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionCut_Video"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Video">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open Video</string>
   </property>
   <property name="shortcut">
    <number>-1</number>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="configure"/>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <number>-1</number>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <number>-1</number>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionCut_Video">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>&amp;Cut Video</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
